docs/help/contributor/style-guide/formatting.md

```diff
@@ -0,0 +1,29 @@
+# Formatting standard and conventions
+
+## Using parentheses
+>Don’t use parentheses, unless it’s the first instance in a page to explain an acronym that isn’t immediately obvious.
+
+>Parenthetical statements especially should be avoided because if something is important enough to be in the sentence, it should be fully part of that sentence.
+
+|:white_check_mark: Correct                  |:no_entry: Incorrect
+|--------------------|-----
+|Install the Custom Resource Definition (CRD) | Check your CLI (you should see it there)
+|Knative Serving will create a `Revision`      | Knative will create a `Revision` (a stateless, snapshot in time of your code and configuration)
```

Comment:
`Revision` here - if this is how we're formatting API objects / resources can that be documented also please.
To me the biggest consistency issue in our docs is formatting these, e.g. channels vs Channels vs `Channels`, services vs Service vs `Services` - let's pick one with the help of eng / the community maybe and move on?
Can we get some user research or input from the TOC on this? Or else just follow the Kubernetes guide but make a firm decision @omerbensaadon ?

Reply:
From a UX perspective, given that Knative is introducing new concepts (via API objects) into a field already rich with concepts and API objects, I would prefer we use `Channels` vs. channels or Channels. 

This allows the user to understand "ah, this is a Knative specific term." 

CC: @RichardJJG @snneji 

Do y'all buy this? I can do a Find + Replace after we've decided

Reply:
If you're asking for a firm decision, this is the decision I would like to make firm

Reply:
I prefer just uppercase, eg "Channels". My reasoning: 
1. I think uppercase is enough to draw attention to word being an API object
2. It seems like overkill to me to use code formatting as well as uppercase
3. The Kubernetes docs seem to use just uppercase for API objects
4. It would add to the list of things in code formatting. If we put all API objects in code it could reduce the impact of other code formatted words. 
5. @omerbensaadon is your suggestion that we treat Knative objects differently from any other Kubernetes objects such as Pods or ConfigMaps? My opinion on this is that I feel like it could be confusing to have Knative objects/concepts in code formatting, but have Kubernetes objects/concepts in normal formatting.

Reply:
@snneji yep that's my basic suggestion, I think you're points are well-taken though. 

I am ok with putting uppercase in the style guide, let's norm on that. 

I'll add it to the Style Guide